chsdi/templates/luftbilder/viewer.mako

```diff
@@ -19,9 +19,7 @@
   displayLink = True
   ## This is a HACK to ensure backward compatibility
   if not layerBodId.startswith('ch.'):
-      templateURL = 'http://' + request.registry.settings['host'] + request.route_path('search', map=topic, _query={'type': 'featuresearch',
-          'features': 'ch.swisstopo.lubis-luftbilder_schwarzweiss,ch.swisstopo.lubis-luftbilder_farbe,ch.swisstopo.lubis-luftbilder-dritte-firmen,ch.swisstopo.lubis-luftbilder-dritte-kantone,ch.swisstopo.lubis-luftbilder_infrarot',
-          'searchText': featureId})
+      templateURL = 'http://' + request.registry.settings['host'] + '/rest/services/' + topic + '/SearchServer?features=ch.swisstopo.lubis-luftbilder_schwarzweiss,ch.swisstopo.lubis-luftbilder_farbe,ch.swisstopo.lubis-luftbilder-dritte-firmen,ch.swisstopo.lubis-luftbilder-dritte-kantone,ch.swisstopo.lubis-luftbilder_infrarot&type=featuresearch&searchText=' + featureId
```

Comment:
We have two issues here, first it should not be HTTP but HTTPS, second the host here is quite ambiguous ! Because now the service search is an external service it would be wiser to add explicitly a new config option: 
```
search_base_url = https://sys-api3.dev.bgdi.ch
```
And to use the ambiguous `host` variable.

Reply:
Good point, thanks!